src/cbc_sdk/connection.py

```diff
@@ -558,7 +558,7 @@ def api_json_request(self, method, uri, **kwargs):
         except ValueError:
             return result
 
-        if "errorMessage" in resp:
+        if "errorMessage" in resp and resp["errorMessage"] is not None:
```

Comment:
Why are you making this change? Given this is `connection.py` this affects the entire SDK